### DISCUSSION TITLE
Is there an available gdg cli command to back up Grafana alert rules and contact points?

### DISCUSSION THREAD

**Discussion by assiakhateeb** [esnet/gdg#312] [Q&A (Help Section)] [answered]

**Comment by safaci2000** [comment]
Not yet, that's in progress.  I have a pending PR that adds contact points.  https://github.com/esnet/gdg/pull/302 WIP.  

There's also https://github.com/esnet/gdg/issues/134 ticket that captures all the entities.  

My only concern with adding this in is that there's a lot of tightly related entities so the user would have to do the restore in order, or potentially have an uber command that restores all alerting entities.

As always PRs are appreciated and accepted if you feel inspired.

**Comment by safaci2000** [comment]
I also dropped support for AlertNotification the legacy version in 0.6.0.  You might get some benefit from trying version 0.5.2 if you like but as Grafana has moved on and it was VERY deprecated when 0.5.2 came out it may not be particularly useful.

**Comment by safaci2000** [comment]
@assiakhateeb I'm converting this to a conversation since it's nothing new tickets/issue wise.  Feel free to close it or chime in as you like.

**Comment by safaci2000** [comment, marked as answer]
I'll call this out as a beta feature but contact points is available.  The format may change before it's finalized since the entire API isn't yet available yet.  The grafana client has an open PR that needs to be fixed and likely the structure will vary based on all the Alerting components and how they're dependent on one another.

Docs:  https://software.es.net/gdg/docs/usage_guide/backup-guide/
